scipy/stats/_relative_risk.py

```diff
@@ -0,0 +1,260 @@
+
+import operator
```

Comment:
```suggestion
from operator import index
```
I think that's all that's being used here, right?

Reply:
Yes, but we almost always use the fully qualified name `operator.index` elsewhere.  I think that is preferable, because `index` sounds fairly generic, and almost looks like it could be a numpy function.

Reply:
I'd agree with that.

Reply:
ok, marking resolved